src/common/utils/MetaKeyUtils.cpp

```diff
@@ -560,6 +560,24 @@ SchemaVer MetaKeyUtils::parseEdgeVersion(folly::StringPiece key) {
          *reinterpret_cast<const SchemaVer*>(key.begin() + offset);
 }
 
+SchemaVer MetaKeyUtils::getLatestEdgeVersion(kvstore::KVIterator* iter, folly::StringPiece& val) {
```

Comment:
Maybe function name `getLatestEdgeVersion` needs to be rename.
Here, not only the maximum version is obtained, but the val corresponding to the maximum version is also obtained.